core/src/main/java/org/elasticsearch/action/support/ToXContentToBytes.java

```diff
@@ -75,6 +76,8 @@ public final String toString() {
             toXContent(builder, EMPTY_PARAMS);
             return builder.string();
         } catch (Exception e) {
+            // So we have a stack trace logged somewhere
+            ESLoggerFactory.getLogger(getClass().getName()).warn("Error building toString", e);;
```

Comment:
does not seem harmless, but not sure we should sneak this in here. Also it may be worth discussing what we do when things break here and what we should do instead.